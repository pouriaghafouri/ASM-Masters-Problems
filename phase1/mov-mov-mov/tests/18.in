7003087320019663845 -4198812583066638522
