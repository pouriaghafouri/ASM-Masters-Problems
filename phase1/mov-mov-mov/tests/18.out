-7244844170623249249
